### PR TITLE
Vet clinic milestone 5: performance

### PR DESCRIPTION
**Pull request by ambrose-kibet**
index
## In this Milestone we:
- [x] Add email column to owners table.
- [x] Add additional data to the DB to test for performance
- [x] Optimized the DB using indexing.
perdormance2
performane
results1
results2
results3

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -50,3 +50,10 @@ CREATE TABLE visits (
   vet_id INTEGER REFERENCES vets(id),
   visit_date DATE
 );
+
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+ CREATE INDEX owners_idx ON owners (id);
+ CREATE INDEX visits_vet_id_idx ON visits (vet_id);
+
```
Comment (1GORDON) [inline schema.sql:58]: Great job team🤝

- [ ] As I can see from the screenshots provided in the PR, only one query `SELECT * FROM visits where vet_id = 2`; used index to improve its performance. The evidence is the index scan in the output.


- [ ] You can use indexes to improve performance for the other two queries (`SELECT COUNT(\*) FROM visits where animal_id = 4`; and `SELECT \* FROM owners where email = 'owner_18327@mail.com'`;). Kindly make sure there is an index scan in the second output the after the improvements with adding indexes. There is no index scan in the outputs of these two commands.

File: schema.sql [hunk 2]
```diff
@@ -50,3 +50,10 @@ CREATE TABLE visits (
   vet_id INTEGER REFERENCES vets(id),
   visit_date DATE
 );
+
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+ CREATE INDEX owners_idx ON owners (id);
+ CREATE INDEX visits_vet_id_idx ON visits (vet_id);
```
Comment (1GORDON) [inline schema.sql:57]: Great job team🤝

- [ ] Kindly add all three created indexes to your schema file. You currently have only two indexes in your schema file. Kindly add the third index.

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @ambrose-kibet & @josuerushanika ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️
- Good commit messages ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi team,

#### Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

### Congratulations! 🎉 

giphy (1)

## (Highlights) Good Points: :+1:
- Your work is well documented. ✔
- The correct workflow is used.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**